packages/cli/test/fixtures/lighthouserc.json

```diff
@@ -22,6 +22,10 @@
       "storage": {
         "sqlDatabasePath": "cli-test-fixtures.tmp.sql"
       }
+    },
+    "wizard": {
+      "serverBaseUrl": "https://github.com/GoogleChrome/lighthouse-ci",
```

Comment:
this is going to be the first collision of property names in the CI file between commands in the config given that they should be identical between them I don't think it's a big deal, but we should track it and try to resolve in next breaking change anyhow

Reply:
Thx, yes I think it should be documented or changed (wizard configuration part).
See my comment above.

Reply:
bump :)